api/github.js

```
@@ -142,116 +157,108 @@ export default async function handler(req) {
-      gfi: null,  // fetched separately via ?gfi=1 to avoid rate limiting
-      ts: Date.now(),
+      activity:  activityDays < 14 ? 'active' : activityDays < 60 ? 'moderate' : 'low',
+      language:  repoData.language,
+      gfi:       null,
+      ts:        Date.now(),
     };
 
     safeCacheSet(repo, result);
-    return new Response(JSON.stringify(result), { status: 200, headers });
-
+    return new Response(JSON.stringify(result), { status: 200, headers: responseHeaders });
```

ESLint

[error] 'responseHeaders' is not defined.

(no-undef)